constantinCurtis/src/main/java/data/collection/task/classes/Item.java

```diff
@@ -1,15 +1,17 @@
 package data.collection.task.classes;
 
 public class Item {
-    public long id;
-    public String title;
-    public Type type;
+    private final long id;
+    private String title;
+    private Type type;
+    private double price;
     private static long idCounter = 0;
 
-    public Item(String title, Type type) {
-        this.id = idCounter++;
+    public Item(String title, Type type, double price) {
+        this.id = ++idCounter;
         this.type = type;
         this.title = title;
+        this.price = price;
```

Comment:
can we have a negative price? I think no? Could you handle this?

Reply:
done, thanks for advice